encoding.bs

```diff
@@ -1331,7 +1324,7 @@ requires buffering of scalar values.
 </dl>
 
 <p>The <dfn constructor for=TextEncoder id=dom-textencoder><code>TextEncoder()</code></dfn>
-constructor, when invoked, must return a new {{TextEncoder}} object.
+constructor, when invoked, must do nothing.
```

Comment:
I think per @domenic it's that we use it for methods (and I was assuming constructors), but not getters/setters. Maybe I got it wrong though.

Reply:
I think I mainly thought "on getting, when invoked" was weird. Otherwise I was probably just going for local consistency. If we wanted a global pattern we should maybe discuss.

Reply:
It's been some time, thoughts on a global pattern?

Reply:
Globally I think it's fairly redundant.

The overall tracking issue is https://github.com/heycam/webidl/issues/730